packages/core/admin/admin/src/content-manager/components/BlocksEditor/Toolbar/index.js

```diff
@@ -160,8 +160,32 @@ const toggleBlock = (editor, value) => {
     // When there is a selection, update the existing block in the tree
     Transforms.setNodes(editor, blockProperties);
   } else {
-    // Otherwise, add a new block to the tree
-    Transforms.insertNodes(editor, { ...blockProperties, children: [{ type: 'text', text: '' }] });
+    /**
+     * When there is no selection, we want to insert a new block just after the last node inserted and prevent the code to add an empty paragraph between them.
+     */
+    const [, lastNodePath] = Editor.last(editor, []);
+    const [parentNode] = Editor.parent(editor, lastNodePath, {
+      match: (node) => node.type !== 'text', // makes sure we get a block node, not an inline node
```

Comment:
Could you move the comment on its own line above the code? Same reason about line length